### DISCUSSION TITLE
wait doesnt work

### DISCUSSION THREAD

**Discussion by Paroryx** [ORelio/Minecraft-Console-Client#1939] [Q&A]
Idk what I did wrong, but the wait in a txt script doesnt work for me :/
`wait 30`
makes the same waiting time as `wait 300` :/
what can I do?

**Comment by ORelio** [comment]
I took a look at the code handling the `wait` command it might happen if you have a space of any unnecessary character in the script line as it defaults to a duration of `10` if it cannot parse the time value.
https://github.com/MCCTeam/Minecraft-Console-Client/blob/95d63183502929cd8ae511afafec19169617153d/MinecraftClient/ChatBots/Script.cs#L202-L210
